tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_stock.feature

```diff
@@ -234,3 +234,53 @@ Feature: Update product combination stock information in Back Office (BO)
       | location                   |       |
       | available date             |       |
     And product "product1" should have no stock movements
+
+  Scenario: I shouldn't be able to add bigger quantity then 2147483647
+    When I update combination "product1MBlack" stock with following details:
+      | delta quantity             | 2147483648  |
+      | minimal quantity           | 1           |
+      | location                   | Storage nr1 |
+      | low stock threshold        | 0           |
+      | low stock alert is enabled | false       |
+      | available date             | 2021-10-10  |
+    Then I should get error that stock available quantity is invalid
+    When I update combination "product1MBlack" stock with following details:
+      | delta quantity             | -2147483649 |
+      | minimal quantity           | 1           |
+      | location                   | Storage nr1 |
+      | low stock threshold        | 0           |
+      | low stock alert is enabled | false       |
+      | available date             | 2021-10-10  |
+    Then I should get error that stock available quantity is invalid
+
+  Scenario: Adding biggest and smallest possible combination quantities
+    When I update combination "product1MBlue" stock with following details:
+      | delta quantity             | 2147483647  |
+      | minimal quantity           | 1           |
+      | location                   | Storage nr1 |
+      | low stock threshold        | 0           |
+      | low stock alert is enabled | false       |
+      | available date             | 2021-10-10  |
+    Then combination "product1MBlue" should have following stock details:
+      | combination stock detail   | value       |
+      | quantity                   | 2147483647  |
+      | minimal quantity           | 1           |
+      | low stock threshold        | 0           |
+      | low stock alert is enabled | false       |
+      | location                   | Storage nr1 |
+      | available date             | 2021-10-10  |
+    When I update combination "product1MBlue" stock with following details:
+      | delta quantity             | -4294967295 |
+      | minimal quantity           | 1           |
+      | location                   | Storage nr1 |
+      | low stock threshold        | 0           |
+      | low stock alert is enabled | false       |
+      | available date             | 2021-10-10  |
+    Then combination "product1MBlue" should have following stock details:
```

Comment:
Can you please add a third test where you edit from the lowest to the max value, please?